eos-toolbox-dev-depends

```diff
@@ -17,7 +17,6 @@ apt-utils
 autoconf
 automake
 bash-completion
-buildah
```

Comment:
Let's remove it from `eos-core-depends` as well since AIUI it's not needed any more? We can always add it back later.

Reply:
Done, updated - also removed from eos-dev-suggests.